plugins/inputs/cloudwatch/README.md

```diff
@@ -68,12 +68,22 @@ API endpoint. In the following order the plugin will attempt to authenticate.
   ## See http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html
   ratelimit = 200
 
+  ## Namespace-wide statistic filters (only gets used if no metrics are defined). These
```

Comment:
I would expect these to be applied to metrics unless overridden on a per-metric level.  Don't mention that these are optional, having them commented out is enough signaling.  Show the default settings:
```
  # statistic_include = ["average", "sum", minimum", "maximum", sample_count"]
  # statistic_exclude = []
```